File: Modul 5/Bahasa Python/PRAK504-2210817110014-MuhammadFiras.py
def reverse(n):
    reversee = 0
    while (n != 0):
        remainder = n % 10
        reversee = reversee * 10 + remainder
        n = n // 10
    return reversee

A,B = map(int, input("").split())
A = reverse(A)
B = reverse(B)
C = A + B
C = reverse(C)
print(C)

File: Modul 5/Bahasa C/PRAK504-2210817110014-MuhammadFiras.c
#include <stdio.h>
int reverse(int n){
    int reversee = 0, remainder;
    while (n != 0) {
        remainder = n % 10;
        reversee = reversee * 10 + remainder;
        n = n / 10;
    }
    return reversee;
}
int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}